### DISCUSSION TITLE
Is there a Swagger API?

### DISCUSSION THREAD

**Discussion by arnabghosh3654** [eclipse-edc/Connector#886] [Q&A] [answered]
Is there a Swagger API to this connector similar to the IDS connector? If yes, how can I access it? The http://localhost:8181/api seems inaccessible (503 error)

**Comment by paullatzelsperger** [comment, marked as answer]
hi. there is an `openapi.yaml` file in `resources/openapi`

**Reply by SebMarsolle** [reply]
For a Java client, do we need to generate the code from openapi.yaml or is there any pre-packaged Java classes ?

**Reply by ndr-brt** [reply]
there's an issue about a generated client https://github.com/eclipse-edc/Connector/issues/1205 that was blocked though.
Now that we have the new management api supporting json-ld there would be the need to have more detailed documentation first. Issue: https://github.com/eclipse-edc/Connector/issues/2871
